app/controllers/inin_chat/application_controller.rb

```diff
@@ -1,4 +1,4 @@
 module IninChat
-  class ApplicationController < ActionController::Base
+  class ApplicationController < ::ApplicationController
```

Comment:
makes inheritance from "original" application controller. As a result we can even interact with devise auth things